basic-auth/README.md

```diff
@@ -0,0 +1,67 @@
+# Basic Authentication
+
+
+This sample allows you to authenticate an incoming request with a Basic Authentication header using a `USER_ID` and a `USER_PASSWORD` as the encoded credential pair. 
```

Comment:
should we mention its not the most secured way of sending but still a mode of security?

Reply:
I'll add a comment in the intro.